back/internal/log/log.go

```diff
@@ -0,0 +1,37 @@
+package log
```

Comment:
If you have the time and feel that it would be worthwhile to practice, please add doc comments for the package, types, and functions.

Reply:
Done